packages/backend/src/managers/SnippetManager.ts

```diff
@@ -21,17 +21,37 @@ import { Request } from 'postman-collection';
 import { Publisher } from '../utils/Publisher';
 import { Messages } from '@shared/Messages';
 import type { RequestOptions } from '@shared/src/models/RequestOptions';
+import { quarkusLangchain4Jgenerator } from './snippets/quarkus-snippet';
+
+type Generator = (requestOptions: RequestOptions) => string;
 
 export class SnippetManager extends Publisher<Language[]> implements Disposable {
+  #languages: Language[];
+  #additionalGenerators: Map<string, Generator>;
```

Comment:
it's not complaining that it can be undefined ? as both of them are not initialized in the constructor but in init() method ?